README.md

```diff
@@ -161,22 +161,36 @@ INFO: For development and testing purposes only
    $ kustomize build config | kubectl apply -f -
    ```
 
-3. Create `cluster-config-v1` configmap to tell the MAO to deploy `kubemark` provider:
+3. Create `cluster` `infrastructure.config.openshift.io` to tell the MAO to deploy `kubemark` provider:
    ```yaml
-   apiVersion: v1
-   kind: ConfigMap
+   apiVersion: apiextensions.k8s.io/v1beta1
+   kind: CustomResourceDefinition
    metadata:
-     name: cluster-config-v1
-     namespace: kube-system
-   data:
-     install-config: |-
-       platform:
-         kubemark: {}
+     name: infrastructures.config.openshift.io
+   spec:
+     group: config.openshift.io
+     names:
+       kind: Infrastructure
+       listKind: InfrastructureList
+       plural: infrastructures
+       singular: infrastructure
+     scope: Cluster
+     versions:
+     - name: v1
+       served: true
+   storage: true
+   ---
+   apiVersion: config.openshift.io/v1
+   kind: Infrastructure
+   metadata:
+     name: cluster
+   status:
```

Comment:
Should this be status or spec?  Seems configurables should be in spec not status.

Reply:
we are just consumers here https://github.com/openshift/installer/blob/master/pkg/asset/manifests/infrastructure.go#L67